src/eddymotion/model.py

```diff
@@ -240,7 +240,7 @@ def predict(self, gradient, **kwargs):
 class AverageDWModel:
     """A trivial model that returns an average map."""
 
-    __slots__ = ("_data", "_gtab", "_th_low", "_th_high", "_bias", "_stat")
+    __slots__ = ("_data", "_th_low", "_th_high", "_bias", "_stat")
 
     def __init__(self, gtab, **kwargs):
```

Comment:
Do you want to also remove the `gtab` input here?

```suggestion
    def __init__(self, **kwargs):
```

Reply:
If yes, don't forget to remove it in the docstring as well.